src/Symfony/Component/Validator/Mapping/ClassMetadata.php

```diff
@@ -262,6 +279,10 @@ public function addConstraint(Constraint $constraint)
      * @param Constraint $constraint The constraint
      *
      * @return ClassMetadata This object
+     *
+     * @throws GroupDefinitionException When the "class name" group has
```

Comment:
If we use the word "default group" in the comment, it will confuse the users. Because they can't tell if we are talking about the "Default" group or the "<ClassName>" group. 
Technically, they are supposed to be identical to each other, but the fact is, if the user explicitly defines "ClassName" group, the issue occurs.